pilot/pkg/xds/xds_test.go

```diff
@@ -371,6 +375,31 @@ spec:
 
 func TestEgressProxy(t *testing.T) {
 	s := NewFakeDiscoveryServer(t, FakeOptions{
+		ObjectString: `
```

Comment:
Shouldn't some check in the test change as well?

May need to create an Endpoint here as well?

Reply:
This was just an experiment from last night/this morning. I'll actually use it in a test before merging.